src/main/steps/review-pay-submit/statement-of-truth/content.test.ts

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable jest/expect-expect */
+import { FormContent, FormFields } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { CommonContent, generatePageContent } from '../../common/common.content';
+
+import { generateContent } from './content';
+
+jest.mock('../../../app/form/validation');
+
+const CY = 'cy';
+const EN = 'en';
+
+const enContent = {
+  section: 'Review your application, pay and send',
+  title: 'Statement of truth',
+  statement:
+    'I understand that proceedings for contempt of court may be brought against anyone who makes, or causes to be made, a false statement in a document verified by a statement of truth without an honest belief in the truth.',
+  reviewBeforeSubmit:
+    "Once you submit your application, you cannot make any further changes. You can select 'Save as draft' to review your application before you submit.",
+  applicant1IBelieveApplicationIsTrue:
+    'I believe that the facts stated in this form and any additional documents are true.',
+  applicant1IBelieveApplicationIsTrue2:
+    'The primary applicant believes that the facts stated in this form and any additional documents are true.',
+  applicant2IBelieveApplicationIsTrue: 'I am authorised by the second applicant to sign this statement.',
+  applicant1SotFullName: 'Enter your full name',
+  applicant2SotFullName: "Enter the second applicant's full name (if applicable)",
+  confirm: 'Confirm',
+  errors: {
+    applicant1IBelieveApplicationIsTrue: {
+      required: 'Confirm your statement of truth',
+    },
```

Comment:
don't we need error when 2nd checkbox is left unticked

Reply:
As per prototype https://ucd-adoption-prototype.herokuapp.com/version-1/check-pay-and-submit/declaration not required.